src/libraries/System.Private.CoreLib/src/System/Type.cs

```diff
@@ -583,5 +567,14 @@ public static Binder DefaultBinder
         public static readonly MemberFilter FilterNameIgnoreCase = (m, c) => FilterNameImpl(m, c!, StringComparison.OrdinalIgnoreCase);
 
         private const BindingFlags DefaultLookup = BindingFlags.Instance | BindingFlags.Static | BindingFlags.Public;
+        // DynamicallyAccessedMemberTypes.All keeps more data than what a member can use, it will keep info about
+        // interfaces and nested types as a whole (the nested type and all its members). Instead this GetAllMembers
```

Comment:
(nit) it might also be good to add a blurb about private base members as well, since that is a key difference.